sklearn/linear_model/tests/test_least_angle.py

```diff
@@ -898,8 +900,8 @@ def test_copy_X_with_auto_gram():
 def test_lars_dtype_match(LARS, has_coef_path, args, dtype):
     # The test ensures that the fit method preserves input dtype
     rng = np.random.RandomState(0)
-    X = rng.rand(6, 6).astype(dtype)
-    y = rng.rand(6).astype(dtype)
+    X = rng.rand(20, 6).astype(dtype)
+    y = rng.rand(20).astype(dtype)
```

Comment:
Is there a reason for this to change?

Reply:
n_samples should be bigger than n_features for `LarsLassoIC` to work